src/OSPSuite.Core/Domain/ParameterIdentifications/OutputMappings.cs

```diff
@@ -46,6 +46,11 @@ public virtual IEnumerable<OutputMapping> OutputMappingsUsingDataRepository(Data
          return All.Where(x => x.UsesObservedData(dataRepository));
       }
 
+      public virtual IEnumerable<DataRepository> AllDataRepositoryMappedTo(string outputFullPath)
+      {
+         return All.Where(x => x.FullOutputPath == outputFullPath).Select(x => x.WeightedObservedData.ObservedData);
```

Comment:
Does it contain the SimulationName? Is this intended?
Also one is called FullOutputPath and the other outputFullPath. Let's make this consistent